openedx_export_plugins/apps.py

```diff
@@ -2,15 +2,22 @@
 """
 openedx_export_plugins Django application initialization.
 """
-
-from __future__ import absolute_import, unicode_literals
-
 from django.apps import AppConfig
 
+from openedx.core.djangoapps.plugins.constants import PluginURLs, ProjectType
+
 
 class OpenedxExportPluginsConfig(AppConfig):
     """
     Configuration for the openedx_export_plugins Django application.
     """
 
     name = 'openedx_export_plugins'
+    plugin_app = {
+        PluginURLs.CONFIG: {
+            ProjectType.CMS: {
+                PluginURLs.NAMESPACE: 'openedx_export_plugins',
+                PluginURLs.RELATIVE_PATH: 'urls',
```

Comment:
@OmarIthawi @bryanlandia  I see this uses `PluginURLs.RELATIVE_PATH` and course access groups and figures uses `PluginURLs.REGEX`. However, they are also plugging into the LMS and not the CMS. Before digging into the plugin code, is `RELATIVE_PATH` common to the LMS and CMS? Is it a new setting post-Hawthorn?

Reply:
I could  remove `PluginURLs.RELATIVE_PATH:` since `urls` is the default.  That's for the dotted path to the urls.py. `PluginURLs.REGEX` was available prior to Juniper, and is for the a URL route prefix, like `r'^api/my_app/'` but I haven't included it because it defaults to `r''` and I've included my prefix `/export/` in my actual urls.py definitions.  I could change that but seems fine to do it either way.  I only have 2 urls so it's not a lot of work to just include it in urls.py  I may fix it in whatever next minor or patch release.

Reply:
I updated

Reply:
Ok, thanks @bryanlandia . I don't know if that is an issue or not. Maybe @OmarIthawi knows better, but I'll approve anyway as I didn't see anything jump out at me to fix